jobs/admin.py

```diff
@@ -29,14 +28,16 @@ def send_status_update_job_offer(modeladmin, request, queryset):
             'jobs/email_templates/status.txt').render(
                 Context({
                     'status': item.get_review_status_display(),
-                    'option': option
+                    'option': option,
+                    'reviewers_comment': item.reviewers_comment,
                 })
         )
         message_html = get_template(
             'jobs/email_templates/status.html').render(
                 Context({
                     'status': item.get_review_status_display(),
-                    'option': option
+                    'option': option,
+                    'reviewers_comment': item.reviewers_comment,
                 })
```

Comment:
move ```Context({...})``` out into a variable so you don't need to duplicate it in the plain and html get_template